UI/src/view/linear_asset/laneModellingForm.js

```diff
@@ -0,0 +1,597 @@
+      };
+
+      return $(title());
+    };
+
+    var createPreviewHeaderElement = function(laneNumbers) {
+      var createNumber = function (number) {
+        var previewButton = $('<td class="preview-lane">' + number + '</td>').click(function() {
+          $(".preview .lane").removeClass("not-highlight-lane highlight-lane").addClass("not-highlight-lane");
+          $(this).removeClass("not-highlight-lane").addClass("highlight-lane");
+          var laneNumber = parseInt(number);
+          lanesAssets.setCurrentLane(laneNumber);
+
+          if(laneNumber.toString()[1] == "1"){
+            currentFormStructure = mainLaneFormStructure;
+          }else{
+            defaultFormStructure.fields[0] = {label: 'Kaista', type: 'read_only_number', publicId: "lane_code", defaultValue: laneNumber, weight: 6};
+            currentFormStructure = defaultFormStructure;
+          }
+          reloadForm($('#feature-attributes'));
+        });
+
+        if(number == lanesAssets.getCurrentLaneNumber()){
+          return previewButton.addClass("highlight-lane");
+        }else{
+          return previewButton.addClass("not-highlight-lane");
+        }
+      };
+
+      var numbers = _.sortBy(laneNumbers);
+
+      var odd = _.filter(numbers, function (number) {
+        return number % 2 !== 0;
+      });
+      var even = _.filter(numbers, function (number) {
+        return number % 2 === 0;
+      });
+
+      var preview = function () {
+        var previewList = $('<table class="preview">');
+
+        var numberHeaders =$('<tr style="font-size: 10px;">').append(_.map(_.reverse(even).concat(odd), function (number) {
+          return $('<th>' + (number.toString()[1] == '1' ? 'Pääkaista' : 'Lisäkaista') + '</th>');
+        }));
+
+        var oddListElements = _.map(odd, function (number) {
+              return createNumber(number);
+            });
+
+        var evenListElements = _.map(even, function (number) {
+          return createNumber(number);
+        });
+
+        return $('<div class="preview-div">').append(previewList.append(numberHeaders).append($('<tr>').append(evenListElements).append(oddListElements))).append('<hr class="form-break">');
+      };
+
+      return preview();
+    };
+
+    var createLaneButtons = function(selectedAsset){
+      var laneAssets = selectedAsset.get();
+      var laneNumbers = _.map(laneAssets, function (lane){
+        return _.head(_.find(lane.properties, function (property) {
+          return property.publicId == "lane_code";
+        }).values).value;
+      });
+
+      var odd =_.filter(laneNumbers, function (number) {
+        return number % 2 !== 0;
+      });
+
+      var even = _.filter(laneNumbers, function (number) {
+        return number % 2 === 0;
+      });
+
+      var addLeftLane = $('<li>').append($('<button class="btn btn-secondary">Lisää kaista vasemmalle puolelle</button>').click(function() {
+        $(".preview .lane").css('border', '1px solid white');
```

Comment:
Things like below it's to be put at CSS files, please.
```javascript
$(".preview .lane").css('border', '1px solid white');

Reply:
Deleted, no need to have that